lexmapr/pipeline.py

```diff
@@ -89,6 +89,7 @@ def find_left_r(s, first, last):
 
 
 # 7-Methods to add suffixes such as food product or product to input phrase to improve Term matching
+# TODO: Does a simple operation like this warrant a function?
```

Comment:
It seems simpler to just do this in-line with `+ " " +` rather than define a function

Reply:
See 505f6eb.